src/main/java/com/exactpro/th2/common/schema/factory/CommonFactory.java

```diff
@@ -202,6 +221,162 @@ public static CommonFactory createFromArguments(String... args) {
+
+        Resource<ConfigMap, DoneableConfigMap> boxConfigMapResource;
+        Resource<ConfigMap, DoneableConfigMap> rabbitMqConfigMapResource;
+        Resource<ConfigMap, DoneableConfigMap> cradleConfigMapResource;
+
+        ConfigMap boxConfigMap;
+        ConfigMap rabbitMqConfigMap;
+        ConfigMap cradleConfigMap;
+
+        Path grpcPath = Path.of(System.getProperty("user.dir") + "/generated_configs/" + ROUTER_GRPC_FILE_NAME);
+        Path rabbitMqPath = Path.of(System.getProperty("user.dir") + "/generated_configs/" + RABBIT_MQ_FILE_NAME);
+        Path cradlePath = Path.of(System.getProperty("user.dir") + "/generated_configs/" + CRADLE_FILE_NAME);
+        Path mqPath = Path.of(System.getProperty("user.dir") + "/generated_configs/" + ROUTER_MQ_FILE_NAME);
+        Path customPath = Path.of(System.getProperty("user.dir") + "/generated_configs/" + CUSTOM_FILE_NAME);
+        Path prometheusPath = Path.of(System.getProperty("user.dir") + "/generated_configs/" + PROMETHEUS_FILE_NAME);
+        Path dictionaryPath = Path.of(System.getProperty("user.dir") + "/generated_configs/dictionary.json");
+
+        try(KubernetesClient client = new DefaultKubernetesClient()) {
+
+            String currentCluster = client.getConfiguration().getCurrentContext().getContext().getCluster();
+
+            boxConfigMapResource = client.configMaps().inNamespace(namespace).withName(boxName + "-app-config");
+            rabbitMqConfigMapResource = client.configMaps().inNamespace(namespace).withName("rabbit-mq-app-config");
+            cradleConfigMapResource = client.configMaps().inNamespace(namespace).withName("cradle");
+
+            boxConfigMap = boxConfigMapResource.require();
+            rabbitMqConfigMap = rabbitMqConfigMapResource.require();
+            cradleConfigMap = cradleConfigMapResource.require();
+
+            ConfigMap dictionaryConfigMap = getDictionary(boxName, client.configMaps().list());
+
+            Map<String, String> boxData = boxConfigMap.getData();
+            Map<String, String> rabbitMqData = rabbitMqConfigMap.getData();
+            Map<String, String> cradleConfigData = cradleConfigMap.getData();
+
+            Service rabbitMqService = client.services().inNamespace("service").withName("rabbitmq-schema").require();
+            Service cassandraService = client.services().inNamespace("service").withName("cassandra-schema").require();
+
+            ObjectNode grpcNode = new ObjectMapper().readValue(boxData.get("grpc.json"), ObjectNode.class);
+            ObjectNode mqNode = new ObjectMapper().readValue(boxData.get("mq.json"), ObjectNode.class);
+            ObjectNode customNode = new ObjectMapper().readValue(boxData.get("custom.json"), ObjectNode.class);
+
+            ObjectNode rabbitMqNode = new ObjectMapper().readValue(rabbitMqData.get("rabbitMQ.json"), ObjectNode.class);
+            ObjectNode cradleNode = new ObjectMapper().readValue(cradleConfigData.get("cradle.json"), ObjectNode.class);
```

Comment:
Use constants instead of magic values.
Use a bean class instead of ObjectNode to parse JSON